packages/ui/components/calendar/src/LionCalendar.js

```diff
@@ -364,7 +364,11 @@ export class LionCalendar extends LocalizeMixin(LitElement) {
     if (this.selectedDate) {
       this.focusSelectedDate();
     } else {
-      this.centralDate = /** @type {Date} */ (this.__initialCentralDate);
+      if (!this.__isEnabledDate(/** @type {Date} */ (this.__initialCentralDate))) {
```

Comment:
Should we call [__calculateInitialCentralDate()](https://github.com/ing-bank/lion/blob/master/packages/ui/components/calendar/src/LionCalendar.js#L381) at the beginning of `initCentralDate` instead? It seems the code is quite similar

Reply:
They do similar things, but there is one big difference:
`__calculateInitialCentralDate()` sets `__initialCentralDate` (when this component is initialized) but this function reads `__initialCentralDate` to reset the `centralDate`.

Actually it seems that the only reason we set `__initialCentralDate` is to reset the `centralDate` when this function is called (when datepicker opens up the calendar view).